app/components/join-steps/status-card.js

```diff
@@ -26,7 +37,46 @@ export default class StatusCardComponent extends Component {
       icon: 'square-check',
     },
   ];
+
+  constructor() {
+    super(...arguments);
+    this.fetchStatus();
+  }
+
+  get status() {
+    return this.args.status || this.fetchedStatus;
+  }
+
+  get feedback() {
+    return this.args.feedback || this.fetchedFeedback;
+  }
+
+  get currentStatusDetails() {
+    return this.APPLICATION_STATUSES.find((s) => s.status === this.status);
```

Comment:
What is this?

Reply:
To get user application status to display over UI

Reply:
okay, saw the user status constant